protos/src/business/v1/business.proto

```diff
@@ -63,8 +63,11 @@ message BusinessMember {
   // The unique identifier of the permission group.
   repeated string permissions_groups_ids = 4;
 
-  // The date since when the
+  // The date since when the user became a member.
   google.protobuf.Timestamp member_since = 5;
+
+  // The unique identifier of the store.
+  optional string store_id = 7;
```

Comment:
Does it mean a business member can only manage one store?

Reply:
no, them can have 1 or more